pallets/handles/src/handles-utils/src/converter.rs

```diff
@@ -0,0 +1,119 @@
+//! # Handle Converter
+//!
+//! `handle_converter` provides a `HandleConverter` struct to detect confusable Unicode characters in a
+//! given input string and return its canonical form.
+use crate::{confusables::CONFUSABLES, types::*};
+use unicode_normalization::{char::is_combining_mark, UnicodeNormalization};
+extern crate alloc;
+use alloc::{
+	string::{String, ToString},
+	vec::Vec,
+};
+
+/// A converter for confusable characters.
+///
+/// Given a string, detects easily confusable characters and returns the string in canonical form.
+pub struct HandleConverter {}
```

Comment:
do we need this struct?

Reply:
This is a remnant of the initial design.

Reply:
Removed.